Library/Homebrew/global.rb

```diff
@@ -30,7 +30,7 @@
 HOMEBREW_USER_AGENT_CURL = ENV["HOMEBREW_USER_AGENT_CURL"]
 HOMEBREW_USER_AGENT_RUBY = "#{ENV["HOMEBREW_USER_AGENT"]} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
```

Comment:
Added in https://github.com/Homebrew/brew/pull/295/commits/b0262f37155ac681ceb119df2e7f852832027bb0